dev/tools/dartdoc.dart

```diff
@@ -50,7 +50,14 @@ Future<void> main(List<String> arguments) async {
   final StringBuffer buf = StringBuffer();
   buf.writeln('name: Flutter');
   buf.writeln('homepage: https://flutter.dev');
-  buf.writeln('version: $version');
+  // TODO(dnfield): We should make DartDoc able to avoid emitting this. If we
+  // use the real value here, every file will get marked as new instead of only
+  // files that have otherwise changed. Instead, we replace it dynamically using
+  // JavaScript.
+  // We'll replace this dynamically. This way, fewer files will get marked as
```

Comment:
"We'll replace this dynamically" is redundant here (already said in the previous sentence)